CIPs/0010.md

```diff
@@ -0,0 +1,84 @@
+---
+cip: 10
+title: General Account Signer Authorization
+author: Nam Chu Hoai (@nambrot), Khoa Phan (@kphed), Eko Mirhard (@eko-mirhard)
+discussions-to: https://github.com/celo-org/celo-proposals/pull/17
```

Comment:
Can you create an issue on this repo for the discussions-to link? that way it doesn't get lost after we merge the PR